reports/config-example.ini

```diff
@@ -0,0 +1,24 @@
+[DEFAULT]
+year = 2021
+month = 12
+month_name = December
+email_from = general+cal-itp@jarv.us
+email_subject = GTFS Quality Reports from Cal-ITP
+email_template = ../templates/email/report.mjml
+is_development = True
+
+[development]
+email_csv_path = test_emails.csv
+show_prompt = True
+postmark_server_token = YOUR_SANDBOX_SERVER_TOKEN
+
+[development2]
+email_csv_path = https://docs.google.com/spreadsheets/d/1AHFa5SKcEn7im374mPwULFCj7VlFchQr/export?gid=289636985&format=csv
```

Comment:
This `email_csv_path` value seems like it should be in the production section. And after moving that, it doesn't seem like this `development2` section is needed and could be removed.

Reply:
my thinking was development1 was using the test CSV and sandbox server whereas development2 is the google doc and the sandbox server. Production would be google doc and sandbox server.

Reply:
I don't really think that extra level of caution is needed. Since there is already a way to bail before seeing the list of emails being sent, I'd vote for just a development and production section. If testing with the sheet is needed, it would be possible to manually set the sandbox token in the production section for a bit anyways.